hwdb/60-keyboard.hwdb

```diff
@@ -88,6 +88,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svneMachines:pneMachines*E725:pvr*
  KEYBOARD_KEY_f3=prog2                                  # "P2" programmable button
  KEYBOARD_KEY_f4=prog1                                  # "P1" programmable button
  KEYBOARD_KEY_f5=presentation
+ KEYBOARD_KEY_f6=power					# Power button
```

Comment:
Please keep indentation characters consistent with the rest of the file. You're using tabs where the original file uses spaces (I have my editor set to mark tabs and trailing white-spaces to catch these errors easily).